lib/Model/Error.php

```diff
@@ -20,6 +20,14 @@
  */
 class Error
 {
+    /**
+     * @param string $message
+     * @param string $class
+     * @param int $code
+     * @param string $file
+     * @param int $line
+     * @param string $trace
+     */
     public function __construct(private $message, private $class, private $code, private $file, private $line, private $trace)
```

Comment:
any reason not to add real type hints here?

Reply:
B/C